SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/Mappings/v1/BrickRec/mapped_v1_dtdlv2_Brick_1_3-REC_4_0.json

```diff
@@ -222,355 +219,355 @@
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Media_Room;1",
       "OutputDtmi": "dtmi:org:w3id:rec:Room;1"
     },
     {
-      "InputDtmi": "dtmi:mapped:core:Medical_Room;1",
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Medical_Room;1",
       "OutputDtmi": "dtmi:org:w3id:rec:HealthcareRoom;1"
     },
     {
-      "InputDtmi": "dtmi:mapped:core:Office;1",
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Office;1",
       "OutputDtmi": "dtmi:org:w3id:rec:Office;1"
     },
     {
-      "InputDtmi": "dtmi:mapped:core:Office_Kitchen;1",
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Office_Kitchen;1",
       "OutputDtmi": "dtmi:org:w3id:rec:Kitchenette;1"
     },
     {
-      "InputDtmi": "dtmi:mapped:core:Open_Office;1",
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Open_Office;1",
       "OutputDtmi": "dtmi:org:w3id:rec:OfficeRoom;1"
     },
     {
-      "InputDtmi": "dtmi:mapped:core:Operating_Mode_Status;1",
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Operating_Mode_Status;1",
       "OutputDtmi": "dtmi:org:brickschema:schema:Brick:Operating_Mode_Status;1"
     },
     {
-      "InputDtmi": "dtmi:mapped:core:Outdoor_Area;1",
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Outdoor_Area;1",
       "OutputDtmi": "dtmi:org:w3id:rec:Space;1"
     },
     {
-      "InputDtmi": "dtmi:mapped:core:Outside;1",
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Outside;1",
       "OutputDtmi": "dtmi:org:w3id:rec:Space;1"
     },
     {
-      "InputDtmi": "dtmi:mapped:core:Parking_Space;1",
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Parking_Space;1",
       "OutputDtmi": "dtmi:org:w3id:rec:ParkingSlotCold;1"
     },
     {
-      "InputDtmi": "dtmi:mapped:core:Parking_Structure;1",
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Parking_Structure;1",
       "OutputDtmi": "dtmi:org:w3id:rec:Garage;1"
     },
     {
       "InputDtmi": "dtmi:mapped:core:Place;1",
       "OutputDtmi": "dtmi:org:w3id:rec:Site;1"
     },
     {
-      "InputDtmi": "dtmi:mapped:core:Plumbing_Room;1",
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Plumbing_Room;1",
       "OutputDtmi": "dtmi:org:w3id:rec:UtilitiesRoom;1"
     },
     {
-      "InputDtmi": "dtmi:mapped:core:Prayer_Room;1",
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Prayer_Room;1",
       "OutputDtmi": "dtmi:org:w3id:rec:MeditationRoom;1"
     },
     {
-      "InputDtmi": "dtmi:mapped:core:Private_Office;1",
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Private_Office;1",
       "OutputDtmi": "dtmi:org:w3id:rec:OfficeRoom;1"
     },
     {
-      "InputDtmi": "dtmi:mapped:core:Pump_Room;1",
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Pump_Room;1",
       "OutputDtmi": "dtmi:org:w3id:rec:UtilitiesRoom;1"
     },
     {
-      "InputDtmi": "dtmi:mapped:core:Reception;1",
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Reception;1",
```

Comment:
Some of these mappings are surprising..  Reception isn't a Brick think according to the REC repo